### PR TITLE
Localised two files from website/layouts to Hindi.

### PR DESCRIPTION
**Pull request by saarques**
Translated file `../404.html` to `../404-hi.html` and `../baseof.html` to `../baseof-hi.html`.

Closes https://github.com/kubernetes/website/issues/13947

### CODE REVIEW

**Comment by mittalyashu** [comment]
@saarques This PR also contains a file from #13916 PR `content/hi/partners/_index.html`, you can remove that file from this PR.

File: layouts/404-hi.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{{ define "main" }}
+<div class="container-404">
+    <h4>
+        शायद आप यह देखना चाहते थे:
```
Comment (mittalyashu) [inline layouts/404-hi.html]: In english, it says `Perhaps you were looking for:`

Doesn't the meaning changes to searching for something in Hindi: `शायद आप यह ढूंढ रहे हैं।`

File: layouts/404-hi.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    </h4>
+
+    <br />
+    <br />
+
+    <ul>
+        <li>
+            <a href="/docs">
+                कुबेरनेट्स डॉक्यूमेंटेशन
```
Comment (mittalyashu) [inline layouts/404-hi.html]: `कुबरनेट्स प्रलेखन`

Reference: https://docs.google.com/spreadsheets/d/1UmL4K0v4yaN34k4WzYDZBobLBp5OZQp4GCTi28AfwQM/edit?usp=sharing

File: layouts/404-hi.html [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+            </a>
+        </li>
+        <li>
+            <a href="/blog">
+                कुबेरनेट्स ब्लॉग
+            </a>
+        </li>
+        <li>
+            <a href="/community">
+                कुबेरनेट्स कम्यूनिटी
```
Comment (mittalyashu) [inline layouts/404-hi.html]: `कुबरनेट्स समुदाय`

Reference: https://docs.google.com/spreadsheets/d/1UmL4K0v4yaN34k4WzYDZBobLBp5OZQp4GCTi28AfwQM/edit?usp=sharing

**Comment by mittalyashu** [comment]
@Rajakavitha1, Do we have to localize these error message too

* https://github.com/kubernetes/website/blob/master/layouts/case-studies/list.html#L51
* https://github.com/kubernetes/website/blob/master/layouts/shortcodes/tabs.html#L35

@saarques Can you localize these files too

* https://github.com/kubernetes/website/blob/master/layouts/partials/templates/errorthrower.html
* https://github.com/kubernetes/website/blob/master/layouts/partials/deprecation-warning.html
* https://github.com/kubernetes/website/blob/master/layouts/partials/feedback.html
* https://github.com/kubernetes/website/blob/master/layouts/partials/footer.html
* https://github.com/kubernetes/website/blob/master/layouts/shortcodes/reference_docs.html
* https://github.com/kubernetes/website/blob/master/layouts/shortcodes/version-check.html

Let @saarques know if there is any other file which requires localization from `website/layouts` directory.

File: layouts/blog/baseof-hi.html [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+		<div class="container-fluid" style="padding-top: 2em">
+			<div class="row">
+				<div class="col-xs-12 col-md-9 text blog-content">
+					{{ block "main" . }}{{ end }}
+				</div>
+				<div class="col-xs-12 col-md-3 text">
+					<div class="widget-content">
+						{{ with site.Home.OutputFormats.Get "rss" -}}
+						<link rel="{{ .Rel }}" type="{{ .MediaType.Type }}" href="{{ .Permalink | safeURL }}" title="{{ site.Title }}">
+						<a class="widget-link" href="{{ .Permalink | safeURL }}"> <i class="fas fa-rss fab-icon"> </i> RSS Feed </a>
```
Comment (mittalyashu) [inline layouts/blog/baseof-hi.html]: RSS Feed => `आरएसएस फ़ीड`

File: layouts/blog/baseof-hi.html [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+						{{ end -}}
+						<a class="widget-link" href="https://kubernetes.io/docs/contribute/start/#write-a-blog-post"><i class="fa fa-edit fab-icon"></i> पोस्ट जमा करें</a>
+						<a class="widget-link" href="https://twitter.com/kubernetesio"> <i class="fab fa-twitter-square fab-icon"> </i> @Kubernetesio</a>
+						<a class="widget-link" href="https://github.com/kubernetes/kubernetes"> <i class="fab fa-github-square fab-icon"></i> Github पर देखें </a>
+						<a class="widget-link" href="http://slack.k8s.io"><i class="fab fa-slack fab-icon"> </i> #कुबेरनेट्स-यूज़र्स </a>
```
Comment (mittalyashu) [inline layouts/blog/baseof-hi.html]: Do not localize the name of slack channels, there is no channel name `#कुबेरनेट्स-यूज़र्स`

File: layouts/blog/baseof-hi.html [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+				<div class="col-xs-12 col-md-9 text blog-content">
+					{{ block "main" . }}{{ end }}
+				</div>
+				<div class="col-xs-12 col-md-3 text">
+					<div class="widget-content">
+						{{ with site.Home.OutputFormats.Get "rss" -}}
+						<link rel="{{ .Rel }}" type="{{ .MediaType.Type }}" href="{{ .Permalink | safeURL }}" title="{{ site.Title }}">
+						<a class="widget-link" href="{{ .Permalink | safeURL }}"> <i class="fas fa-rss fab-icon"> </i> RSS Feed </a>
+						{{ end -}}
+						<a class="widget-link" href="https://kubernetes.io/docs/contribute/start/#write-a-blog-post"><i class="fa fa-edit fab-icon"></i> पोस्ट जमा करें</a>
```
Comment (mittalyashu) [inline layouts/blog/baseof-hi.html]: Can someone suggest a better word for this?

File: layouts/blog/baseof-hi.html [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+						<a class="widget-link" href="https://twitter.com/kubernetesio"> <i class="fab fa-twitter-square fab-icon"> </i> @Kubernetesio</a>
+						<a class="widget-link" href="https://github.com/kubernetes/kubernetes"> <i class="fab fa-github-square fab-icon"></i> Github पर देखें </a>
+						<a class="widget-link" href="http://slack.k8s.io"><i class="fab fa-slack fab-icon"> </i> #कुबेरनेट्स-यूज़र्स </a>
+						<a class="widget-link" href="https://stackoverflow.com/questions/tagged/kubernetes"><i class="fab fa-stack-overflow fab-icon"></i> Stack Overflow</a>
+						<a class="widget-link" href="https://discuss.kubernetes.io"><i class="fab fa-discourse fab-icon"></i>फोरम</a>
+						<a class="widget-link" href="https://kubernetes.io/docs/setup/pick-right-solution/"><i class="fa fa-download fab-icon"></i> कुबेरनेट्स डाउनलोड करें</a>
```
Comment (mittalyashu) [inline layouts/blog/baseof-hi.html]: Update the spelling of Kubernetes `कुबरनेट्स`

Reference: https://docs.google.com/spreadsheets/d/1UmL4K0v4yaN34k4WzYDZBobLBp5OZQp4GCTi28AfwQM/edit?usp=sharing

**Comment by Rajakavitha1** [comment]
> @Rajakavitha1, Do we have to localize these error message too
>  Lets take a look at other localized files to see if this was localized. 
May be @raelga can let us know what they did for spanish?

> * https://github.com/kubernetes/website/blob/master/layouts/case-studies/list.html#L51
> * https://github.com/kubernetes/website/blob/master/layouts/shortcodes/tabs.html#L35
> 
> @saarques Can you localize these files too
> 
> * https://github.com/kubernetes/website/blob/master/layouts/partials/templates/errorthrower.html
> * https://github.com/kubernetes/website/blob/master/layouts/partials/deprecation-warning.html
> * https://github.com/kubernetes/website/blob/master/layouts/partials/feedback.html
> * https://github.com/kubernetes/website/blob/master/layouts/partials/footer.html
> * https://github.com/kubernetes/website/blob/master/layouts/shortcodes/reference_docs.html
> * https://github.com/kubernetes/website/blob/master/layouts/shortcodes/version-check.html
> 
> Let @saarques know if there is any other file which requires localization from `website/layouts` directory.

File: layouts/404-hi.html (hunk 1 above)
Comment (saarques) [reply layouts/404-hi.html]: I just minded the tenses here, "देखना चाहते थे" for "were looking for". I can change it if you want.

File: layouts/404-hi.html (hunk 2 above)
Comment (saarques) [reply layouts/404-hi.html]: Okay, will change this.

File: layouts/404-hi.html (hunk 3 above)
Comment (saarques) [reply layouts/404-hi.html]: Okay.

File: layouts/blog/baseof-hi.html (hunk 5 above)
Comment (saarques) [reply layouts/blog/baseof-hi.html]: Okay, sure!

File: layouts/blog/baseof-hi.html (hunk 7 above)
Comment (saarques) [reply layouts/blog/baseof-hi.html]: Sure.

**Comment by saarques** [comment]
@mittalyashu How to remove this file `content/hi/partners/_index.html` from this PR?

File: layouts/404-hi.html (hunk 1 above)
Comment (mittalyashu) [reply layouts/404-hi.html]: @Rajakavitha1 any suggestions on this one.

Comment (Rajakavitha1) [reply layouts/404-hi.html]: how about: शायद आप यह खोज रहे हैं

Comment (saarques) [reply layouts/404-hi.html]: Sounds good to me, will change in next commit.

File: layouts/blog/baseof-hi.html (hunk 6 above)
Comment (Rajakavitha1) [reply layouts/blog/baseof-hi.html]: How about? ब्लॉग जमा करें

Comment (saarques) [reply layouts/blog/baseof-hi.html]: We can do it, if "post" and "blog" are meant to be same here, as the original English statement was "Submit Post".

Comment (Rajakavitha1) [reply layouts/blog/baseof-hi.html]: we don't need to do an exact word to word translation. we have to convey the intent of the English text in Hindi.

Comment (saarques) [reply layouts/blog/baseof-hi.html]: Okay, sure, no problem, changing this.

**Comment by mittalyashu** [comment]
@raelga Can you shed some light on this comment?

https://github.com/kubernetes/website/pull/13950#issuecomment-487263510

**Comment by saarques** [comment]
> Can you review these changes @saarques?

Seems like I have changed legitimate ones @mittalyashu , requested some changes from @Rajakavitha1 . 
After that we can have another go to the review round.

**Comment by Rajakavitha1** [comment]
> > Can you review these changes @saarques?
> 
> Seems like I have changed legitimate ones @mittalyashu , requested some changes from @Rajakavitha1 .
> After that we can have another go to the review round.

I am done with the review. After you incorporate the changes we can have one more review and get an /lgtm and /approve. Thanks for this PR !!!

**Comment by saarques** [comment]
> > > Can you review these changes @saarques?
> > 
> > 
> > Seems like I have changed legitimate ones @mittalyashu , requested some changes from @Rajakavitha1 .
> > After that we can have another go to the review round.
> 
> I am done with the review. After you incorporate the changes we can have one more review and get an /lgtm and /approve. Thanks for this PR !!!

All the changes are already done @Rajakavitha1 , we are ready for the final review.

**Comment by mittalyashu** [comment]
We are still left with an update from @raelga regarding https://github.com/kubernetes/website/pull/13950#issuecomment-487263510

**Comment by mittalyashu** [comment]
@saarques Can you extend the scope of this PR to full `layouts` directory?
If so, change the title to `Localised layouts directory to Hindi`

**Comment by saarques** [comment]
> @saarques Can you extend the scope of this PR to full `layouts` directory?
> 
> If so, change the title to `Localised layouts directory to Hindi`

Hey @mittalyashu sorry for late response, I will do it after some time, I am busy with college projects and internship as of now.

**Comment by mittalyashu** [comment]
Okay.

**Pull request closed without merging**